airflow/bin/cli.py

```diff
@@ -283,6 +292,43 @@ def _tabulate(pools):
         log.info(_tabulate(pools=pools))
 
 
+def pool_import_helper(filepath):
+    with open(filepath, 'r') as poolfile:
+        pl = poolfile.read()
+    try:
+        d = json.loads(pl)
+    except Exception:
+        print("Invalid pool file.")
+    else:
+        try:
+            pools = []
+            n = 0
+            for k, v in d.items():
+                if isinstance(v, dict) and len(v) == 2:
+                    pools.append(api_client.create_pool(name=k,
+                                                        slots=v["slots"],
+                                                        description=v["description"]))
+                    n += 1
+                else:
+                    pass
+        except Exception:
+            pass
+        finally:
+            print("{} of {} pool(s) successfully updated.".format(n, len(d)))
+            return pools
+
+
+def pool_export_helper(filepath):
+    pool_dict = {}
+    pools = api_client.get_pools()
+    for pool in pools:
+        pool_dict[pool[0]] = {"slots": pool[1], "description": pool[2]}
```

Comment:
This code makes me a bit sad, I would expect a more descriptive response from the API.

Reply:
I totally felt the same way, however, since it returns https://github.com/apache/incubator-airflow/blob/da1f64cf805abaecda227fc8c43a15a92faff648/airflow/api/client/local_client.py#L42

The pool class doesn't seem have set and get as Variables, maybe it's good to have another PR to add them?
https://github.com/apache/incubator-airflow/blob/da1f64cf805abaecda227fc8c43a15a92faff648/airflow/models.py#L5077

Reply:
Yes, that would be my preference.

Reply:
@Fokko Do you think we can move forward PR or this needs to wait for adding set and get in Pool? Thanks!